selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/core/CheSeleniumSuiteModule.java

```diff
@@ -78,44 +66,33 @@ public void configure() {
     bind(TestSvnUsernameProvider.class).to(CheTestSvnUsernameProvider.class);
     bind(TestSvnRepo1Provider.class).to(CheTestSvnRepo1Provider.class);
     bind(TestSvnRepo2Provider.class).to(CheTestSvnRepo2Provider.class);
-    bind(TestWorkspaceUrlResolver.class).to(CheTestWorkspaceUrlResolver.class);
-    bind(TestUserNamespaceResolver.class).to(CheTestUserNamespaceResolver.class);
+
     bind(TestApiEndpointUrlProvider.class).to(CheTestApiEndpointUrlProvider.class);
     bind(TestIdeUrlProvider.class).to(CheTestIdeUrlProvider.class);
     bind(TestDashboardUrlProvider.class).to(CheTestDashboardUrlProvider.class);
 
-    bind(HttpJsonRequestFactory.class).to(TestDefaultUserHttpJsonRequestFactory.class);
-    install(new FactoryModuleBuilder().build(TestUserHttpJsonRequestFactoryCreator.class));
-
-    bind(TestAuthServiceClient.class).to(CheTestAuthServiceClient.class);
-    bind(TestMachineServiceClient.class).to(CheTestMachineServiceClient.class);
-
-    bind(TestUser.class).to(TestUserImpl.class);
     bind(TestWorkspaceProvider.class).to(TestWorkspaceProviderImpl.class).asEagerSingleton();
 
+    install(new FactoryModuleBuilder().build(TestUserHttpJsonRequestFactoryCreator.class));
     install(new FactoryModuleBuilder().build(TestWorkspaceServiceClientFactory.class));
+    install(new FactoryModuleBuilder().build(TestUserFactory.class));
 
-    install(
-        new FactoryModuleBuilder()
-            .implement(TestUser.class, TestUserImpl.class)
-            .build(TestUserFactory.class));
-
-    bind(AdminTestUser.class).to(CheAdminTestUser.class);
+    if (parseBoolean(System.getenv(CHE_MULTIUSER_VARIABLE))) {
```

Comment:
I can see the same bindings in both CheSeleniumMultiUserModule and CheSeleniumSingleUserModule.
Let's keep them in CheSeleniumSuiteModule